src/components/App.js

```diff
@@ -0,0 +1,12 @@
+import '../App.css';
+import Calculator from './Calculator';
+
+function App() {
+  return (
+    <div className="App">
+      <Calculator />
+    </div>
+  );
+}
+
```

Comment:
- [ ] For this project, it's expected your App.js component should be class-based. Kindly refactor this component to be a class-based component. 👍🏽